client/src/locales/en-US.json

```diff
@@ -1165,5 +1165,26 @@
         "createAccount": "Create an account",
         "createAccountUrl": "https://parsec.cloud/en/tarification/",
         "forgottenPasswordLink": "https://parsec.cloud/password_recovery"
+    },
+    "SidebarPage": {
```

Comment:
There's a mismatch between translation files, the english one has `SidebarPage.*` whereas the french has `ClientApplication.SidebarPage.*`